backend/graphql/internal/schema/dataloaders/README.md

```diff
@@ -0,0 +1,5 @@
+1) Get the generator `go get github.com/vektah/dataloaden`
```

Comment:
dataloader caches the data, or just batches and optimizes the calls to backend/uac?
we only do it for uac because we anticipate only those calls to have the volume which makes sense to batch?

Reply:
Just batches and optimizes if the same call happens over and over, like what happens on collaborators (teams and users).

We don't cache anything between calls as permissions and state might have changed. Caching is a nightmare.